.github/workflows/misc-approve-and-merge-pr.yml

```
@@ -0,0 +1,54 @@
+# Merge Update Version PR Workflow
+# This workflow automates the process of merging and approving a pull request for updating the version of a repository.
+
+name: Merge Update Version PR
+
+on:
+  workflow_call:
+    inputs:
+      pull-request-number:
+        required: true
+        type: string
+      repository:
+        required: true
+        type: string
+
+jobs:
+  approve-and-merge-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Displaying the input values for reference
+      - name: Log inputs
+        run: |
+          {
+            echo "# Inputs"
+            echo "| Key                 | Value                             |"
+            echo "| ------------------- | --------------------------------- |"
+            echo "| pull-request-number | ${{ inputs.pull-request-number }} |"
+            echo "| repository          | ${{ inputs.repository }}          |"
+          } >> $GITHUB_STEP_SUMMARY
```

La oss teste dette for å fikse shellcheck

```suggestion
          } >> "$GITHUB_STEP_SUMMARY"
```